contents/tutorials/css-selectors-for-actions.md

```diff
@@ -0,0 +1,171 @@
+- Products from different categories (you can represent categories via data attributes)
+- Different HTML tags which you want to combine into one action (again using data attributes)
+- UI states represented with pseudo-classes like `:hover`, `:visited` and others
+- Targeting one specific element by id (e.g anchors)
+
+> As a rule of thumb, it is probaly a **bad idea** to use complex CSS selectors with class names and combinators. Layouts tend to change, class names can be unreliable (especially if you use Tailwind CSS). So if you are considering using CSS selectors, **data attributes** is probably the best option to control specificity.
+
+
+## Configuring CSS selectors
+There are two ways to do it - through the Toolbar and Data Management section.
+
+### Toolbar
+Go to your PostHog instance and select `Toolbar` in the bottom of the sidebar.
+
+![Toolbar Location](..images/tutorials/../../../images/tutorials/css-selectors-for-actions/toolbar_location.png)
+
+Then, press the `Open with Toolbar` button which will take you to your website. You can read more about working with Toolbar [here](docs/tutorials/../../toolbar.md).
+
+After your website is open, you should see a PostHog logo 🦔 floating around. Press it. You will see a toolbar like on a picture below.
+
+![Toolbar In Action](..images/tutorials/../../../images/tutorials/css-selectors-for-actions/toolbar_in_action.png).
+
+The best option to start creating actions with CSS selectors is to use `Inspect` option in the toolbar and then click on some element on the page. You should see the menu with a `Create a new action` button in the bottom. Press it and you will see the `Actions` menu with automatically generated CSS selector in the `Selector` section. Feel free to start from there and change it!
+
+Alternatively, you can go to the `Actions` option in the toolbar, select `New action` and then press `Select Element`. You should get the `Actions` menu like a one below:
+
+![Action Menu](..images/tutorials/../../../images/tutorials/css-selectors-for-actions/create_action.png).
+
+Tick the `Selector` checkbox and you are ready to create CSS selectors!
+
+### Data Management section
+Another way to create CSS selectors is to go to the `Data Management` section which is located in the sidebar of your PostHog instance.
+
+![Data Management Location](..images/tutorials/../../../images/tutorials/css-selectors-for-actions/data_management_location.png).
+
+On the `Data Management` page, select the `Actions` tab. Then press the `New Action` button. You should see the menu like this:
+
+![Create New Action Data Management](..images/tutorials/../../../images/tutorials/css-selectors-for-actions/create_new_action_data_management.png).
+
+Now you can select the `Inspect element` option, but it will eventually bring you back to the `Toolbar` which we covered earlier. So instead of this we wiil select `From event or pageview` option.
```

Comment:
```suggestion
Now you can select the `Inspect element` option, but it will eventually bring you back to the `Toolbar` which we covered earlier. So, instead we wiil select `From event or pageview` option.
```